### PR TITLE
Fix script path to include working directory and script name

### LINKED ISSUE #5373: Cannot debug the python file in opened folder
VS Version: 16.2.0 Preview 4.0 [29013.205.d16.2stg]
Python Tool Version: 16.2.19150.1

Repro step:
1. Create a folder
2. Open this folder in VS
3. Add a python file
5. Press F5 

Actual Result:
Error message popup after pressing F5
image

Expect Result:
It should no error popup

### PR DESCRIPTION
**Pull request by RaymonGulati1**
Fix #5373

### CODE REVIEW

File: Python/Product/Workspace/PythonLaunchDebugTargetProvider.cs [hunk 1]
```diff
@@ -134,7 +134,7 @@ class PythonLaunchDebugTargetProvider : ILaunchDebugTargetProvider {
             var launchConfig = new LaunchConfiguration(config) {
                 InterpreterPath = config == null ? path : null,
                 InterpreterArguments = settings.GetValue(InterpreterArgumentsKey, string.Empty),
-                ScriptName = scriptName,
+                ScriptPath = Path.Combine(workingDir, scriptName),
```
Comment (huguesv) [inline Python/Product/Workspace/PythonLaunchDebugTargetProvider.cs]: We should only do this if it's not already a full path.
You can check with `Path.IsPathRooted`

Comment (RaymonGulati1) [reply Python/Product/Workspace/PythonLaunchDebugTargetProvider.cs]: Fixed

**Pull request merged**